README.md

```diff
@@ -1,2 +1,219 @@
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello Microverse <a name="about-project"></a>
+
+> 
+
+**Hello Microverse** is a...
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **Linters SetUp**
+- **Follow HTML and CSS best practice**
+- **Display Text**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](git@github.com:KUBAHO3/Hello-Microverse.git)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- Visual Studio Code.
+- Node JS.
+- Git bash.
+- GitHub Account.
+
+<!--
+Example command:
+```sh
+ gem install rails
+```
+ -->
+### Setup
+
+Clone this repository to your desired folder:
+Use git clone command or download ZIP folder
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:KUBAHO3/Hello-Microverse.git
+```
+--->
+### Install
+Install this project with:
+npm
+Example command:
+```sh
+  cd my-project
+  npm init -y
+```
+### Usage
+To run the project, execute the following command:
+npm start or live server
+Example command:
+```sh
+  GitHub Pages Server
+```
+### Run tests
+To run tests, run the following command:
+npm test
+Example command:
+```sh
+  npx stylelint "**/*.{css,scss}"
+```
+### Deployment
+You can deploy this project using:
+GitHub Pages
+Example:
+```sh
+git@github.com:KUBAHO3/Hello-Microverse.git
+```
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- AUTHORS -->
+## :busts_in_silhouette: Authors <a name="authors"></a>
+:bust_in_silhouette: **Gladwin Tshepo Ramantso**
+- GitHub: [@KUBAHO3](https://github.com/KUBAHO3)
+- Twitter: [@KubahoH](https://twitter.com/KubahoH)
+- LinkedIn: [Kubaho Linne Heaven](https://www.linkedin.com/in/kubaho-linne-heaven-78ab37208/)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- FUTURE FEATURES -->
+## :telescope: Future Features <a name="future-features"></a>
+- [ ] **making it responsive**
+- [ ] **Adding Animation**
+- [ ] **Making it dynamic**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Please give more attention to the formatting of this section. See the image below as the bullet points are on one line instead of multiple lines.
![image](https://user-images.githubusercontent.com/84156714/218977419-d683fd4c-3b86-4de5-8837-bf82ea2364ab.png)

Reply:
resolved